example.js

```diff
@@ -0,0 +1,32 @@
+// API proposal for Redux rest library
+
+// just usage examples
+
+import { rest } from '../lib/redux_api';
+
+// issue a get request
+const { user: { username }} = SETTINGS;
+rest.dashboard.get(username);
+
+rest.profile.get(username).then(profile => {
```

Comment:
It looks like this allows multiple verbs per reducer. What about two uses of the same verb? I don't think we have any APIs that use that right now, but theoretically we could implement `/api/v0/programs/`  and `/api/v0/programs/1/` which would both use GET and should put their information in the same reducer. But maybe that's YAGNI for now